archinstall/lib/installer.py

```diff
@@ -237,7 +241,7 @@ def generate_key_files(self):
 			gen_enc_file = self._disk_encryption.should_generate_encryption_file(part_mod)
 
 			luks_handler = Luks2(
-				part_mod.dev_path,
```

Comment:
This is the only one I'm a bit worried about, but it's probably fine? `dev_path` can cause an exception, and that's maybe something we want to happen?

Reply:
I looked at the code and at this point the partition must probably have a device path

Reply:
In `guided.py` this is true, as the call comes after some sanity checks and stuff.
But if they were to copy this and run it super quick after the formatting step - I wonder if it might not properly report the right data?
We don't have any sanity checks in `Luks2` directly so this might get dicy?